runtime/src/main/java/org/corfudb/runtime/exceptions/TransactionAbortedException.java

```diff
@@ -2,6 +2,7 @@
 
 import lombok.Getter;
 import org.corfudb.protocols.wireprotocol.TxResolutionInfo;
```

Comment:
I am not a fan of the existing exception construction. It takes dependencies on classes deep in corfu. It feels a little odd since packages will have cyclic dependencies. Transactions will depend on exceptions and exceptions will depend on transactions.